presto-main/src/test/java/com/facebook/presto/sql/planner/planPrinter/TestPlanPrinter.java

```diff
@@ -186,4 +189,29 @@ public void testDomainTextFormatting()
                 Domain.onlyNull(VARCHAR),
                 "[NULL]");
     }
+
+    @Test
+    public void testPrintNativeExecutionNode()
+    {
+        ValuesNode values = PLAN_BUILDER.values();
+        LimitNode limit = PLAN_BUILDER.limit(10, values);
```

Comment:
Can we instead build a valid simple mapper plan here (like scan->limit->blahblah)? Test code will be used for references by developers and they will try to understand the usage by looking at the tests sometimes. So it would be great to have tests having stuff that will actually appear in real scenarios for better illustration purposes.